plugins/source-verifier/profile.json

```diff
@@ -2,13 +2,16 @@
     "name": "source-verification",
     "displayName": "Source Verifier",
     "description": "Source metadata fetcher and validator",
-    "version": "0.1.7-beta",
+    "version": "0.2.0-beta",
```

Comment:
In case you have some documentation, you can put a link here. The key is `documentation`

Reply:
Currently closest to documentation for source verify is medium article which in my opinion is not the best resource to put here.